.ci/Jenkinsfile

```diff
@@ -534,11 +534,14 @@ def generateFunctionalTestStep(Map args = [:]){
     goArch = "arm64"
   }
 
+  def githubCheckSha1 = params.GITHUB_CHECK_SHA1?.trim() ? params.GITHUB_CHECK_SHA1 : ''
+  def githubCheckRepo = params.GITHUB_CHECK_REPO?.trim() ? params.GITHUB_CHECK_REPO : ''
+
   // Setup environment for platform
   def envContext = []
   envContext.add("PROVIDER=${provider}")
-  envContext.add("GITHUB_CHECK_SHA1=${env.GITHUB_CHECK_SHA1}")
-  envContext.add("GITHUB_CHECK_REPO=${env.GITHUB_CHECK_REPO}")
+  envContext.add("GITHUB_CHECK_SHA1=${githubCheckSha1}")
+  envContext.add("GITHUB_CHECK_REPO=${githubCheckRepo}")
```

Comment:
IIUC, those env variables were already in place as per:
- https://github.com/elastic/e2e-testing/blob/742ab4bacb3ed53c45cf34a841e636e831ed3fc3/.ci/Jenkinsfile#L86-L87